x/adminmodule/types/message_submit_proposal.go

```diff
@@ -23,25 +28,71 @@ func NewMsgSubmitProposal(content Content, proposer sdk.AccAddress) (*MsgSubmitP
 	return m, nil
 }
 
-//func (m *MsgSubmitProposal) GetContent() Content {
```

Comment:
Maybe we should remove these lines ?

Reply:
We're currently use workaround to explicitly unmarshall content.types.Any to concrete structs, which is suboptimal.
We now investigate the problem, this commented code helps us to figure out the problem.